src/script/page/AppMain.tsx

```diff
@@ -203,10 +226,8 @@ export const AppMain: FC<AppMainProps> = ({
       '/preferences/av': () => mainView.list.openPreferencesAudioVideo(),
       '/preferences/devices': () => mainView.list.openPreferencesDevices(),
       '/preferences/options': () => mainView.list.openPreferencesOptions(),
-      '/user/:userId(/:domain)': (userId: string, domain: string = apiContext.domain ?? '') => {
-        showMostRecentConversation();
-        showUserModal({domain, id: userId}, () => navigate('/'));
-      },
+      '/user/:userId/:domain': showUserProfile,
+      '/user/:userId': showUserProfile,
```

Comment:
If we want to support the domain before the userId, I'd add it here - it would be clear for others that there's a case like that.

```suggestion
      '/user/:userId/:domain': showUserProfile,
      '/user/:userId': showUserProfile,
      'user/:domain/:userId': showUserProfile
```